config/application.rb

```diff
@@ -17,6 +17,8 @@ class Application < Rails::Application
     # Settings in config/environments/* take precedence over those specified here.
     # Application configuration should go into files in config/initializers
     # -- all .rb files in that directory are automatically loaded.
+
+    config.eager_load_paths += %W[#{config.root}/lib]
```

Comment:
coul there be a comment explaining why we want this, and why we want it for all environments?  Or is it super obvious and I don't have the rails foo?